application-admintools-ui/src/main/resources/AdminTools/HelpLinks.xml

```diff
@@ -39,75 +41,52 @@
 {{/box}}
 
 {{velocity}}
+= $escapetool.xml($services.localization.render('adminTools.helppage.performance.title')) =
 
-= Performance issues =
-
-{{html}}
-  &lt;ul>
-    &lt;li>&lt;a href="https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HSizing" target="_blank">
-      Hardware requirements&lt;/a>&lt;/li>
-    &lt;li>&lt;a href="https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HMemory" target="_blank">
-      Memory recommendations&lt;/a>&lt;/li>
-    &lt;li>&lt;a href="https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/Performances/#HDocumentCache"
-      target="_blank">Cache recommendations&lt;/a>&lt;/li>
-  &lt;/ul>
-{{/html}}
-
-= Server issues =
+* [[$escapetool.xml($services.localization.render('adminTools.helppage.performance.hardware'))
```

Comment:
Same as above. There's no HTML/XML context AFAICS. And if the translation doesn't have parameters it is way simpler and clearer to use the translation macro (no need for Velocity).